src/tests/ftest/erasurecode/multiple_failure.py

```diff
@@ -64,7 +61,7 @@ def test_ec_multiple_targets_on_same_rank(self):
         :avocado: tags=all,full_regression
         :avocado: tags=hw,large,ib2
```

Comment:
only if you need to push again - ib2 can be removed here too

Reply:
Done